pkgs/by-name/ut/utpm/package.nix

```
@@ -5,19 +5,19 @@
   openssl,
   pkg-config,
 }:
-rustPlatform.buildRustPackage {
+rustPlatform.buildRustPackage rec {
```

prefer finalAttrs over rec (allows for a bit more convenient recursion)
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
(and a closing bracket at the end)